docs/app.css

```diff
@@ -15,25 +23,9 @@ body {
 .remark-slide-content {
   background-color: #000;
   font-size: 2rem;
-  background-image: radial-gradient(
-      circle at 100% 10%,
-      #ffffff11 10%,
-      #000000 60%
-    ),
-    repeating-linear-gradient(
-      0deg,
-      #ffffff22,
-      #ffffff22 0.5px,
-      #0000 1px,
-      #0000 27px
-    ),
-    repeating-linear-gradient(
-      90deg,
-      #ffffff22,
-      #ffffff22 0.5px,
-      #0000 1px,
-      #0000 70px
-    );
```

Comment:
If we are converting it to an image, we should add only a slice of the image and repeat it.